src/swarm-topography/Swarm/Game/Scenario/Topography/Rasterize.hs

```diff
@@ -5,13 +5,15 @@
 module Swarm.Game.Scenario.Topography.Rasterize where
 
 import Codec.Picture
+import Data.List.NonEmpty qualified as NE
 import Data.Vector qualified as V
 import Swarm.Game.Scenario.Topography.Area
 
 -- | Converts linked lists to vectors to facilitate
 -- random access when assembling the image
 gridToVec :: Grid a -> V.Vector (V.Vector a)
```

Comment:
I think this function should move to `Swarm.Game.Scenario.Topography.Area` (especially if we stop exporting the constructors of `Grid`).